src/pages/index.tsx

```diff
@@ -1,177 +1,570 @@
-  }
-];
+const JupiterTopics = () => {
+  return (
+    <div className="mt-10 xl:mt-24">
+      <div className="text-2xl font-semibold tracking-tighter  text-center max-md:max-w-full">
+        Quick Start
+      </div>
 
-const tweetsFeatures: TweetCardProps[] = [
-  {
-    author: {
-      name: "Quirky Qwerty",
-      username: "QuirkyQwerty_",
-      image:
-        "https://pbs.twimg.com/profile_images/1514622423387226118/NreOjFMM_400x400.jpg",
-    },
-    tweet: `Jupiter's DCA function changes the shitcoining game.
+      <div className="mt-4 w-full grid grid-cols-2 md:grid-cols-3 justify-center gap-2 text-sm xl:text-base font-bold whitespace-nowrap flex-wrap px-2 lg:px-0">
+        {/* <a href="" className="p-3 lg:px-5 lg:py-4 rounded-2xl bg-lime-200/[.03] hover:bg-lime-200/10 max-md:text-center !text-lime-200 hover:!no-underline">
+          Welcome to Solana
+        </a> */}
+        <a
+          href="https://station.jup.ag/blog/green-paper"
+          className="p-3 lg:px-5 lg:py-4 rounded-2xl bg-lime-200/[.03] hover:bg-lime-200/10 max-md:text-center !text-lime-200 hover:!no-underline"
+        >
+          Jupiter Green Paper
+        </a>
+        <a
+          href="#section-lfg-launchpad"
+          className="p-3 lg:px-5 lg:py-4 rounded-2xl bg-lime-200/[.03] hover:bg-lime-200/10 max-md:text-center !text-lime-200 hover:!no-underline"
+        >
+          LFG Launchpad
+        </a>
 
-    Makes sol shitters charts look vastly different from every other chain. No spooky wicks down, no TP-enticing wicks up. 
-    
-    Slow and smooth.
-    
-    One direction only.`,
-    time: new Date(2023, 11, 22, 13, 24),
-    permalink: "https://twitter.com/QuirkyQwerty_/status/1738067850366275968",
-  },
-  {
-    author: {
-      name: "Joe Kreate",
-      username: "joekreate",
-      image:
-        "https://pbs.twimg.com/profile_images/1648119841189150722/-j2p8UeW_400x400.jpg",
-    },
-    tweet:
-      `Think we take how amazing @JupiterExchange is for granted. Can't tell you how nice it is to have on-chain limit orders and DCA. 
-      Don't even think I realized that's what kept me from doing more on-chain trading until it existed.`,
-    time: new Date(2023, 11, 16, 10, 27),
-    permalink: "https://twitter.com/joekreate/status/1735848992679076161",
-  },
-  {
-    author: {
-      name: "Kulture ⚡",
-      username: "KultureElectric",
-      image:
-        "https://pbs.twimg.com/profile_images/1682323094365511680/dhpZVpna_400x400.jpg",
-    },
-    tweet: `I have so much admiration for @weremeow and the @JupiterExchange team 🙏
-    
-    Truly Solana aligned and leading the way in transparent and honest communication.
-    
-    Not to mention that LFG Launchpad actually worked super well and will kick-start a new era of fair token launches`,
-    time: new Date(2024, 1, 1, 14, 5),
-    permalink: "https://twitter.com/KultureElectric/status/1752936182785130819",
-  },
-];
+        <a
```

Comment:
i think we should use `target=_blank` for this, so it opens another tab.

Reply:
this is done, along with ref rel

Reply:
this is done, along with ref rel